src/main/kotlin/com/bugsnag/android/gradle/BugsnagUploadProguardTask.kt

```diff
@@ -82,7 +82,8 @@ open class BugsnagUploadProguardTask @Inject constructor(
         val manifestInfo = parseManifestInfo()
 
         // Send the request
-        val request = BugsnagMultiPartUploadRequest.from(this)
+        val newEndPoint = endpoint.get() + SUBDOMAIN_PROGUARD
```

Comment:
I think we should maybe ensure that the `endpoint` URL is in the expected format. Something like:

```suggestion
        val newEndPoint = endpoint.get().removeSuffix("/") + SUBDOMAIN_PROGUARD
```

Just in case the URL already ended with a `/`